webpack.requirejs_jquery_q.config.js

```diff
@@ -0,0 +1,25 @@
+var path = require('path');
```

Comment:
Curious - what is the difference in output size between our old build and this new build? Wondering diff between almond and whatever webpack is using.